jena-arq/src/main/java/org/apache/jena/riot/tokens/TokenizerText.java

```diff
@@ -384,7 +378,31 @@ private Token parseToken() {
             case CH_RBRACE:     reader.readChar(); token.setType(TokenType.RBRACE);    /*token.setImage(CH_RBRACE);*/ return token;
 
             case CH_LPAREN:     reader.readChar(); token.setType(TokenType.LPAREN);    /*token.setImage(CH_LPAREN);*/ return token;
-            case CH_RPAREN:     reader.readChar(); token.setType(TokenType.RPAREN);    /*token.setImage(CH_RPAREN);*/ return token;
+
+            // Can be ')' or ')>>'
```

Comment:
@afs what's the document that describes the `<<(` and `)>>` grammar? I'm only finding "<<" and ">>" as in RDF-star?

Reply:
`<<(`, `)>>` are RDF 1.2 and not in RDF-star CG (community group). RDF 1.2 has `<<`,`>>` - very long story.  
https://www.w3.org/TR/rdf12-turtle/

I hope that sometime this will settle down enough to 

These tokens L_TRIPLE and R_TRIPLE tokens, are not used in any parser in this PR. 
This is part of the overlap with RDF 1.2. work.

Reply:
Ah, there it is! Thanks Andy!